src/otelcollector/otelcol-config.yml

```diff
@@ -19,3 +23,7 @@ service:
       receivers: [otlp]
       processors: [batch]
       exporters: [logging, jaeger]
+    metrics:
+      receivers: [otlp]
+      processors: []
```

Comment:
Should the `batch` processor be specified here as a good practice for all pipelines?

Reply:
I think you are right, the [doc](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/batchprocessor) of batch processor said that:

> It is highly recommended to configure the batch processor on every collector.

Reply:
Done